src/en/infinityscans/src/eu/kanade/tachiyomi/extension/en/infinityscans/WebviewInterceptor.kt

```diff
@@ -55,7 +55,7 @@ class WebviewInterceptor(private val baseUrl: String) : Interceptor {
 
             webview.webViewClient = object : WebViewClient() {
                 override fun shouldInterceptRequest(view: WebView?, request: WebResourceRequest?): WebResourceResponse? {
-                    if (request?.url.toString().contains("$baseUrl/ajax/turnstile")) {
+                    if (request?.url.toString().contains("$baseUrl/api/verification")) {
```

Comment:
Can you do this without converting to string? I'm assuming `contains` is used to detect the path regardless of query parameters.

```kt
request?.url.encodedPath == "/api/verification"
```

Reply:
I don't know why exactly it is checked to that way.
I only updated the requested url to match the reality.

Should I change it, as it now works.

Reply:
It's not critical, so leaving it as-is is also valid.